client/post-editor/editor-sidebar/header.jsx

```diff
@@ -77,15 +77,18 @@ function EditorSidebarHeader( { typeSlug, type, siteId, showDrafts, toggleDrafts
 
 export default connect(
 	( state ) => {
-		const siteId = get( getSelectedSite( state ), 'ID' );
+		const site = getSelectedSite( state );
+		const siteId = get( site, 'ID' );
 		const postId = getEditorPostId( state );
+		const isJetpack = get( site, 'jetpack' );
 		const typeSlug = get( getEditedPost( state, siteId, postId ), 'type' );
 
 		return {
 			siteId,
 			typeSlug,
 			type: getPostType( state, siteId, typeSlug ),
-			showDrafts: isEditorDraftsVisible( state )
+			showDrafts: isEditorDraftsVisible( state ),
+			isJetpack
```

Comment:
One of the big conveniences with `react-redux` `connect` is being able to avoid passing down state through props, but rather `connect` immediately on the component which requires the data. This is to say, I think we could introduce `connect` to the `<EditorDraftsButton />` component and supply `jetpack` as a prop there.

Aside being I think we ought to (and are conventionally as of late) avoid `is` as a prefix for props, so this be named `jetpack` instead.